specs/latest/2.0/index.html

```diff
@@ -807,60 +807,60 @@ <h2 class="no-toc">Table of contents</h2>
                                optional GLuint srcOffset = 0);
 
   /* Programs and shaders */
-  [WebGLHandlesContextLoss] GLint getFragDataLocation(WebGLProgram? program, DOMString name);
+  [WebGLHandlesContextLoss] GLint getFragDataLocation(WebGLProgram program, DOMString name);
 
   /* Uniforms */
-  void uniform1ui(WebGLUniformLocation? location, GLuint v0);
-  void uniform2ui(WebGLUniformLocation? location, GLuint v0, GLuint v1);
-  void uniform3ui(WebGLUniformLocation? location, GLuint v0, GLuint v1, GLuint v2);
-  void uniform4ui(WebGLUniformLocation? location, GLuint v0, GLuint v1, GLuint v2, GLuint v3);
+  void uniform1ui(WebGLUniformLocation location, GLuint v0);
```

Comment:
Same here; these APIs must allow the WebGLUniformLocation to be nullable, and continue to be a no-op if it's null.